cli/plasmo/src/features/helpers/crypto.ts

```diff
@@ -0,0 +1,9 @@
+import { createHash } from "crypto"
+
+/**
+ * Fast hash for local file revving
+ * DO NOT USE FOR SENSITIVE PURPOSES
+ * md5 is good enough for file-revving: https://github.com/sindresorhus/rev-hash
```

Comment:
we can even do crc32 for file diff if we want to squeeze out perf

Reply:
I think even google was struggling at implementing crc32 for their storage lol